src/test/url-handling.test.js

```diff
@@ -169,7 +169,7 @@ describe('url upgrading', function() {
     it('can upgrade callTreeFilters to transforms', function() {
       const { getState } = _getStoreWithURL({
         search:
-          '?callTreeFilters=prefix-012~prefixjs-123~postfix-234~postfixjs-345',
+          '?callTreeFilters=prefix-012~prefixjs-123~postfix-234~postfixjs-345&thread=0',
```

Comment:
Call tree filters need to know which thread they are being applied to. Before it defaulted to 0, which this thread will only not be defined in tests, not in end-user code.